woocommerce/woocommerce
# CYS: allow to the user to move the pattern

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR allows the user to move blocks/patterns. 

https://github.com/woocommerce/woocommerce/assets/4463174/527c27ce-e9ba-454d-88f2-59ed8bd5420a



Closes #47325, #47326 and https://github.com/woocommerce/woocommerce/issues/47570



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure that you are using a Gutenberg trunk version
2. Make sure you have the WooCommerce beta tester plugin installed
3. Head over to Tools > WCA Test Helper > Features
4. Make sure you see the `pattern-toolkit-full-composability` on the list.
5. Enable it.
4. Head over to WooCommerce > Home > Customize your store.
5. Click on "Start designing" and proceed to the Pattern Assembler.
6. Click on "Design your Homepage".
7. Switch Pattern
8. Ensure that the block toolbar appears and you can move the block.
9. Ensure that you will not be able to move the block before the header and after the footer.




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

CYS: allow to the user to move the pattern.

#### Comment 



</details>
